docs/frontend.md

```diff
@@ -0,0 +1,34 @@
+# Frontend Architecture Overview
+
+The Common LIMS frontend app code can be found in `src/static/sentry`. Javascript tests are written in Jest and can be found in `tests/js/spec`.
+
+## Components
+
+Common LIMS uses React for its frontend components.
+
+## Asynchronous calls
```

Comment:
Would API calls perhaps be better here? (even if the API calls happen to be asynchronous).

Reply:
Fixed